.github/actions/setup/action.yml

```diff
@@ -7,30 +7,23 @@ runs:
       uses: pnpm/action-setup@v2
       with:
         version: 8.6.5
-
     - name: Setup node 16.x
       uses: actions/setup-node@v3
       with:
         node-version-file: .nvmrc
         registry-url: https://registry.npmjs.org
         cache: pnpm
 
-    - name: Get foundry version
-      id: foundry-version
-      run: echo ::set-output name=version::$(cat .foundryrc)
-
     - name: Setup foundry
       uses: foundry-rs/foundry-toolchain@v1
-      with:
-        version: $${ steps.foundry-version.outputs.version }
```

Comment:
foundry deletes images from time to time so this broke. In future we should consider using CI builder image here